src/pipeline/tasks/03_train_model.py

```
@@ -0,0 +1,57 @@
+import pandas as pd
+import xgboost as xgb
+import os
+import joblib
+from sklearn.metrics import log_loss, accuracy_score, classification_report
```

Ruff (0.12.2)

`sklearn.metrics.classification_report` imported but unused

Remove unused import: `sklearn.metrics.classification_report`

(F401)